packages/functional-tests/pages/resetPassword.ts

```diff
@@ -71,13 +71,17 @@ export class ResetPasswordPage extends BaseLayout {
   }
 
   get newPasswordHeading() {
-    return this.page.getByRole('heading', { name: 'Create new password' });
+    return this.page.getByRole('heading', { name: 'Create a new password' });
   }
 
   get newPasswordTextbox() {
     return this.page.getByRole('textbox', { name: 'New password' });
   }
 
+  get newPasswordLabel() {
```

Comment:
Should this be `newPasswordInput`?

Reply:
It's the container around the input. The way it's structured is a bit funn since the input is nested. Maybe `newPasswordInputContainer` would be better?